nanodbc/nanodbc.cpp

```diff
@@ -3439,6 +3439,59 @@ void result::result_impl::get_ref_impl(short column, T& result) const
 namespace nanodbc
 {
 
+std::list<datasource> list_dsns()
```

Comment:
I'd suggest we avoid abbreviations and use clear self-descriptive names, i.e. `list_datasources`

Reply:
Thanks, will change.

Reply:
Done - let me know if I missed anything.

Reply:
Thanks